compiler/src/iree/compiler/GlobalOptimization/test/raise_special_ops.mlir

```diff
@@ -538,3 +538,42 @@ util.func public @conv_nchw_extf_both(%arg0 : tensor<1x5x10x10xf16>,
 //  CHECK-SAME:     %[[ARG1:.+]]: tensor<5x5x3x3xf16>
 //       CHECK:   %[[RESULT:.+]] = linalg.conv_2d_nchw_fchw {{.*}} ins(%[[ARG0]], %[[ARG1]]
 //       CHECK:   util.return %[[RESULT]]
+
+// -----
+
+util.func public @jit_eval_168(%arg0: tensor<1024x7x7x2xi8>) -> tensor<1024x7x7xi8> {
+  %0 = tensor.empty() : tensor<1024x7x7x2xf32>
+  %cst = arith.constant 5.000000e-01 : f32
+  %1 = linalg.generic {indexing_maps = [affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>, affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>], iterator_types = ["parallel", "parallel", "parallel", "parallel"]} ins(%arg0 : tensor<1024x7x7x2xi8>) outs(%0 : tensor<1024x7x7x2xf32>) {
+  ^bb0(%in: i8, %out: f32):
+    %4 = arith.extsi %in : i8 to i32
+    %5 = arith.sitofp %4 : i32 to f32
+    %6 = arith.mulf %5, %cst : f32
+    linalg.yield %6 : f32
+  } -> tensor<1024x7x7x2xf32>
+
+  %extracted_slice = tensor.extract_slice %1[0, 0, 0, 1] [1024, 7, 7, 1] [1, 1, 1, 1] : tensor<1024x7x7x2xf32> to tensor<1024x7x7xf32>
+  %2 = tensor.empty() : tensor<1024x7x7xi8>
+  %cst_0 = arith.constant 0.000000e+00 : f32
+  %cst_1 = arith.constant -1.280000e+02 : f32
+  %cst_2 = arith.constant 1.270000e+02 : f32
+
+
+  %3 = linalg.generic {indexing_maps = [affine_map<(d0, d1, d2) -> (d0, d1, d2)>, affine_map<(d0, d1, d2) -> (d0, d1, d2)>], iterator_types = ["parallel", "parallel", "parallel"]} ins(%extracted_slice : tensor<1024x7x7xf32>) outs(%2 : tensor<1024x7x7xi8>) {
+  ^bb0(%in: f32, %out: i8):
+    %4 = arith.divf %in, %cst : f32
+    %5 = math.round %4 : f32
+    %6 = arith.addf %5, %cst_0 : f32
+    %7 = arith.cmpf ult, %6, %cst_1 : f32
+    %8 = arith.cmpf ugt, %6, %cst_2 : f32
+    %9 = arith.select %7, %cst_1, %6 : f32
+    %10 = arith.select %8, %cst_2, %9 : f32
+    %11 = arith.fptosi %10 : f32 to i8
+    linalg.yield %11 : i8
+  } -> tensor<1024x7x7xi8>
+  util.return %3 : tensor<1024x7x7xi8>
```

Comment:
Can you remove everything except for the `tensor.extract_slice` from this test, given that you are only checking for the transpose + slice?